.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -43,15 +66,23 @@ message += `\n\n platform | result \n ---|--- \n🤖 android 🤖|${androidResul
 message += `\n🍎 iOS 🍎|${iOSResult} \n🕸 web 🕸|${webResult}`;
 
 /**
- * Create comment on each pull request
+ * Comment Single PR
+ *
+ * @param {Object} pr
+ * @returns {Promise<void>}
  */
-prList.forEach((pr) => {
-    githubUtils.createComment(github.context.repo.repo, pr, message, octokit)
+function commentPR(pr) {
+    return githubUtils.createComment(GitHub.context.repo.repo, pr, message, octokit)
```

Comment:
I see that it was there before, but the `octokit` parameter is unnecessary / unused on this function. `GithubUtils` takes an octokit instance in its constructor.

Reply:
oh Yeah. I didn't notice.